src/Sulu/Bundle/SecurityBundle/Admin/SuluSecurityContactsContentNavigationProvider.php

```diff
@@ -2,30 +2,36 @@
 
 namespace Sulu\Bundle\SecurityBundle\Admin;
 
-use Sulu\Bundle\AdminBundle\Navigation\ContentNavigationInterface;
+use Sulu\Bundle\AdminBundle\Navigation\ContentNavigationProviderInterface;
 use Sulu\Bundle\AdminBundle\Navigation\ContentNavigationItem;
 use Sulu\Component\Security\Authorization\SecurityCheckerInterface;
 
-class SuluSecurityContactsContentNavigation implements ContentNavigationInterface
+class SuluSecurityContactsContentNavigationProvider implements ContentNavigationProviderInterface
```

Comment:
I'll keep it like this in this PR, and create a separate one for dropping the prefixes in all content navigation and admin classes.

So `SuluSecurityContactsContentNavigationProvider` becomes `ContactsContentNavigationProvider` and `SuluSecurityAdmin` becomes `SecurityAdmin`.